src/util/storybook-docs/Table.tsx

```diff
@@ -42,13 +54,11 @@ export const TableCell = ({ children, isCSS, copy, ...props }) => {
   );
 };
 
-TableCell.propTypes = {
-  children: PropTypes.node,
-  copy: PropTypes.bool,
-  isCSS: PropTypes.bool,
-};
+export interface TableHeadLayoutProps {
+  headers?: Array<string | number>;
```

Comment:
Is there a reason we would choose to use `Array<string | number>` vs `string | number[]` or (string | number)[]? Would we want to standardize our approach here?

Reply:
there's no difference,  `Type[]` is the shorthand for `Array<Type>`, maybe in just scenarios it would be easier to ready `Array<Array<Type>>` than `Type[][]`.

example:
![image](https://user-images.githubusercontent.com/69323251/147125905-e3f8a40e-45ee-4963-adae-ccc338dbc7f1.png)

if we're making a decision i'd say i'm leaning a bit more towards `(string | number)[]`.